docs/blog/mapsicle-2-1.html

```
@@ -0,0 +1,116 @@
+<h1>Mapsicle 2.1: an MPL-2.0 object mapper, and honest numbers against AutoMapper</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)